tests/mock_crust/client_restrictions.rs

```diff
@@ -151,17 +152,26 @@ fn resend_parts_on_exceeding_limit() {
     }
     poll_and_resend(&mut nodes, &mut clients);
 
-    // `MAX_PARTS / 2` parts will be resent from client.
-    let expect_sent_parts = (2 * MAX_PARTS) as u64;
-    assert_eq!(
-        clients[0].inner.get_user_msg_parts_count(),
-        expect_sent_parts
+    // NOTE: these calculations are somewhat approximate – in reality, the user message
+    // parts seem to be around 20,363 bytes each, not 20,480, so an extra one can
```

Comment:
Awesome, that works well. I've left a note in case this calculation breaks with changes to serialisation or message content.